crates/router/src/configs/settings.rs

```diff
@@ -759,6 +760,18 @@ impl Settings<SecuredSecret> {
     }
 }
 
+impl Settings<RawSecret> {
```

Comment:
Can we implement these methods on the `KvConfig` instead?

Reply:
then the caller should have to handle non kv_store feature as well?